bin/index.js

```diff
@@ -0,0 +1,134 @@
+    const templatePath = path.join(parentDir, 'templates',template.name);
+    const destinationPath = path.join(targetDir);
+
+    const initSpinner = createSpinner('Running npm init...').start()
+    try {
+      // execSync('npm init -y', { stdio: 'inherit', cwd: targetDir });
+      execSync('npm init -y', { stdio: 'ignore', cwd: targetDir });
+      initSpinner.success({ text: 'npm init completed successfully.' });
+    } catch (error) {
+      initSpinner.error({ text: `Error running npm init:\n` });
+      console.error(error.message);
+      return;
+    }
+
+    //console.log(`Copying server template from ${templatePath} to ${destinationPath}`);
+
+    const copySpinner = createSpinner('Creating server files...').start();
+    try {
+      fs.copySync(templatePath, destinationPath);
+      copySpinner.success({text:"Created server files successfully."});
+    } catch (error) {
+      copySpinner.error({text:"Error creating server files.\n"})
+      console.error(error.message);
+    }
+
+      
+    const addDependencies = createSpinner("Adding dependency packages...").start();
+    try {
+      // Update package.json to add express as a dependency
+      const packageJsonPath = path.join(targetDir, 'package.json');
+      const packageJsonContent = fs.readFileSync(packageJsonPath, 'utf8');
+      const packageJson = JSON.parse(packageJsonContent);
+      packageJson.dependencies = packageJson.dependencies || {};
+      // packageJson.dependencies.express = "^4.17.1";
+      template.dependencies.forEach((dependency)=>{
+        packageJson.dependencies[`${dependency.name}`] = dependency.version;
+      });
+      fs.writeFileSync(packageJsonPath, JSON.stringify(packageJson, null, 2));
+
+      addDependencies.success({text:'Added dependency packages successfully.'});
+    } catch(error) {
+      addDependencies.error("Error adding dependency packages.\n");
+      console.error(error.message);
+    }
+      
+    
+    const installDependencies = createSpinner("Installing dependency packages...").start();
+    try {
+      //execSync('npm i', { stdio: 'inherit', cwd: targetDir });
+      execSync('npm i', { stdio: 'ignore', cwd: targetDir });
+      
+      installDependencies.success({text:'Installed dependencies successfully.'});
+
+      console.log(chalk.green.bold('\nSetup complete! To run your server:'));
+      console.log(chalk.yellow('Run:'), chalk.white.bold('npm start'));
+    } catch (error) {
+      installDependencies.error({text:'Error installing dependencies.\n'})
+      console.error(error);
+    }
+  });
+
+program
+  .command('clear')
+  .description('Clear the directory')
+  .action(()=>{
+    const targetDir = process.cwd();
+    console.log('Clearing Directory...',chalk.bgRed.white(targetDir));
+    const clearingDirectory = createSpinner('Deleting All Files').start();
+    try {
+      // Read the directory
+      const files = fs.readdirSync(targetDir);
+      
+      // Remove each file/directory except . and ..
```

Comment:
Is this comment relevant now that you've commented out the logic it says.
If you still wish to keep it, enclose `.` and `..` in `''` and end the comment with a full stop.